ssz-rs/src/error.rs

```diff
@@ -45,6 +48,7 @@ impl std::error::Error for Error {}
 
 #[derive(Debug)]
 pub enum TypeError {
```

Comment:
shall we add a doc comment here? open to exact phrasing
```suggestion
/// An invalid type
pub enum TypeError {
```